orchagent/fdborch.h

```diff
@@ -5,29 +5,70 @@
 #include "observer.h"
 #include "portsorch.h"
 
+enum FdbOrigin
+{
+    FDB_ORIGIN_INVALID = 0,
+    FDB_ORIGIN_LEARN = 1,
+    FDB_ORIGIN_PROVISIONED = 2,
+    FDB_ORIGIN_VXLAN_ADVERTIZED = 4
+};
+
 struct FdbEntry
 {
     MacAddress mac;
     sai_object_id_t bv_id;
-    std::string port_name;
 
     bool operator<(const FdbEntry& other) const
     {
         return tie(mac, bv_id) < tie(other.mac, other.bv_id);
     }
+    bool operator==(const FdbEntry& other) const
+    {
+        return tie(mac, bv_id) == tie(other.mac, other.bv_id);
+    }
 };
 
 struct FdbUpdate
 {
     FdbEntry entry;
     Port port;
+    string type;
     bool add;
 };
 
+struct FdbData
```

Comment:
This structure will contain all the information related to an FDB entry, so, it would be neat keep VxLAN related info in a seperate structure and add it here as a variable, but I will leave it up to you!

Reply:
keeping it the same for now.